### ISSUE TITLE
MenuBar: ARIA attribute is not allowed: aria-level="1"

### ISSUE THREAD

**Issue by ali-418** [primefaces/primevue#7249]
### Describe the bug

Reopen of https://github.com/primefaces/primevue/issues/6401. The issue was closed with the `Cannot Replicate` label but the issue is still present.

Quoted from https://github.com/primefaces/primevue/issues/6401:

> [_aria-level_ property](https://www.w3.org/TR/wai-aria-1.2/#aria-level) can be used with roles: [heading](https://www.w3.org/TR/wai-aria-1.2/#heading), [listitem](https://www.w3.org/TR/wai-aria-1.2/#listitem), [row](https://www.w3.org/TR/wai-aria-1.2/#row), or [treeitem](https://www.w3.org/TR/wai-aria-1.2/#treeitem). The `<li>` elements used to create the menu items use the [_menuitem_ role](https://www.w3.org/TR/wai-aria-1.2/#menuitem). This role does not allow for the _aria-level_ attribute.

This is also flagged by the Accessibility Insights for Web tool. ([aria-allowed-attr](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr))

![Image](https://github.com/user-attachments/assets/f416b3cd-e5e1-4875-a980-55c1ea7da16b)

![Image](https://github.com/user-attachments/assets/c877bd6f-041a-417c-a68a-b71be4243738)

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/dj98txgi

### Environment

PrimeVue 4 and Nuxt 3. However, the issue is also present on the PrimeVue website (https://primevue.org/menubar/)

### Vue version

3

### PrimeVue version

4.2.5

### Node version

20.17.0

### Steps to reproduce the behavior

1. Go to the reproducer
2. Open the preview in a new tab and connect to project (this is important for the next step)
3. Run the [Accessibility Insights for Web](https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni?hl=en) extension and select the FastPass option
4. The extension reports error for [aria-allowed-attr](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr) ([WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html))

### Expected behavior

Accessibility Insights for Web reports no error for aria-allowed-attr

**Comment by tugcekucukoglu** [comment]
References: 
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role
- https://www.w3.org/WAI/ARIA/apg/patterns/menubar/
